### ISSUE TITLE
[GM] (MARLO) Update rules for indicator fields

### ISSUE THREAD

**Issue by Sagsso** [CCAFS/MARLO#2270]
Fields to update according to the guidance & SMO Feedback

POLICIES 
- [ ] Description - Update to mandatory

OICR
- [ ] Title - Update word limit to 35

INNOVATIONS
- [ ] Evidence - Must allow only one link. Other links should be additional evidence.

MILESTONES
- [ ] Evidence link - Update to **Mandatory** if status is "Changed" & **Optional** if status is "Cancelled" OR "Extended"
- [ ] Cross-cutting markers (_Gender, Youth, CapDev, Climate Change_) justification mandatory if marked 1.Significant OR 2.Principal

**Comment by gcmartinezg** [comment]
@htobon could you confirm we have to enable multiple geographic scopes to be chosen for innovations? @Sagsso and I were searching on the excel sheet of the SMO changes proposed for AR2021 and we were not able to find that requirement. The only thing similar to it was a comment from the MEL team asking if it was necessary to add a "multinational" option, and we already can do it on MARLO.

**Comment by htobon** [comment]
Yes, I can confirm that the idea would be to allow multiple geographic scopes for the innovations, the same way we are doing it for other indicators like OICRs and Policies.
